### DISCUSSION TITLE
CLI API

### DISCUSSION THREAD

**Discussion by karthikmallireddy** [duckdb/duckdb#2619] [General]
HI,

When will be the CLI API will be available

**Comment by Mytherin** [comment]
What do you mean with CLI API? The CLI is available for download [here](https://duckdb.org/docs/installation/).

**Comment by karthikmallireddy** [comment]
I was referring to the below link


https://duckdb.org/docs/api/cli

**Reply by Mytherin** [reply]
Please file an issue in [the documentation repo](https://github.com/duckdb/duckdb-web) for documentation related manners.

**Reply by karthikmallireddy** [reply]
Thanks Mytherin I just submitted it.
